org.civicrm.reqangular/src/common/services/hr-settings.js

```diff
@@ -0,0 +1,12 @@
+define([
+    'common/angular',
+    'common/modules/services'
+], function (angular, module) {
+    'use strict';
+
+    module.factory('HR_settings', function(){
```

Comment:
Sorry, I remembered you could alter constants, but apparently the overriding value is applied only in the component where the override happens